src/Sylius/Bundle/PromotionBundle/Doctrine/ORM/PromotionRepository.php

```diff
@@ -80,6 +80,7 @@ protected function filterByActive(QueryBuilder $queryBuilder, \Datetime $date =
                     $queryBuilder->expr()->gt($this->getPropertyName('endsAt'), ':date')
                 )
             )
+            ->andWhere($this->getPropertyName('enabled') . ' = 1')
```

Comment:
That was my first idea too, until I saw that all other fields use `getPropertyName` too.